provider/pkg/clients/clients.go

```diff
@@ -316,12 +316,20 @@ func IsSecret(obj *unstructured.Unstructured) bool {
 	return (gvk.Group == corev1.GroupName || gvk.Group == "core") && gvk.Kind == string(kinds.Secret)
 }
 
-// IsConfigMap returns true if the resource is a configmap marked as immutable.
```

Comment:
This is technically unrelated but given that this method doesn't actually consider whether the object is immutable or not it felt misleading, so I figured I'd get rid of it.